roles/swift-ring/handlers/main.yml

```diff
@@ -1,5 +1,19 @@
 ---
-- name: setup swift rings
+- name: setup swift rings - bootstrap
   command: sudo -u swiftops /usr/local/bin/swifttool -i /home/swiftops/.ssh/id_rsa
-           bootstrap --config /etc/swift/ring_definition.yml --outdir /etc/swift
+           --config {{ swift_ring.outdir }}/ring_definition.yml --outdir {{ swift_ring.outdir }} bootstrap
```

Comment:
The -u argument is being passed to sudo, which is saying run as the `swiftops` user. In Ansible, that's accomplished by the `become_user` argument, which tells Ansible which user to sudo to.